tests/foreman/performance/common/perf_candlepin.py

```diff
@@ -0,0 +1,94 @@
+from robottelo.common import conf, ssh
+
+import logging
+import time
+import requests
+
+class CandlepinPerfTest:
+    """
+    Measures performance of RH Satellite 6 - Candlepin Subscription functionality
+    """
+    username = conf.properties['foreman.admin.username']
+    password = conf.properties['foreman.admin.password']
+    serverhost = conf.properties['main.server.hostname']
+
+    # create logger
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
+    requests.packages.urllib3.disable_warnings()
+
+    '''
+    Subscribe VM to Satellite by Register + ActivationKey
+    '''
+    @classmethod
+    def single_register_activation_key(cls, ak_name, default_org, vm):
+        # note: must create ssh keys for vm if running on local
+        # result = ssh.command('subscription-manager clean', hostname=vm)
+        result = ssh.command('subscription-manager clean', hostname=vm)
+        if result.return_code != 0:
+           cls.logger.error('Failed to clean sub-mgr!')
+           cls.logger.error(result.stderr)
+
+        result = ssh.command('time -p subscription-manager register --activationkey={0} --org={1}'.format(ak_name, default_org), hostname=vm)
+        # if result.return_code == 0:
+        real_time = [real for real in result.stderr.split('\n') if real.startswith('real')]   
```

Comment:
Below, I see that `real_time[0]` is being selected. Is there any reason to execute a list comprehension and then select only the first element in that list? I suspect a regex or `''.split()` would be a more elegant one-line solution.

Can you provide a simple example of what the text being worked with looks like? I'm not looking for a five-line piece that dwarfs the code, just something simple like:

# e.g. "foo\nreal bar\nreal biz\nbaz"
return my_string_splitting_code